internal/cli/tenant/update.go

```diff
@@ -198,17 +197,4 @@ func (o *UpdateOptions) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&o.ConnectWhiteList, FLAG_CONNECT_WHITE_LIST, "", "The connect white list of ob tenant")
 	cmd.Flags().StringToStringVar(&o.ZonePriority, FLAG_ZONE_PRIORITY, nil, "zone priority config of OBTenant")
 	cmd.Flags().BoolVarP(&o.force, FLAG_FORCE, "f", DEFAULT_FORCE_FLAG, "force operation")
-	o.AddUnitFlags(cmd)
-}
-
-// AddUnitFlags add unit-resource-related flags
-func (o *UpdateOptions) AddUnitFlags(cmd *cobra.Command) {
```

Comment:
Why was this function deleted?

Reply:
The previous commit was incomplete and is now committed. In the `update` command of tenant, i want to provide the user with fewer flags for a clearer description.